tests/behat/uploadFileToMoodle.feature

```diff
@@ -43,3 +43,30 @@ Feature: upload the resource in oCIS to moodle
     And I click on "//*[@class='fp-filename-field']/p[text()='testfile.txt']" "xpath_element"
     And I click on "Select this file" "button"
     Then I should see "testfile.txt"
+
+  Scenario: click refresh button to get latest resource on Personal drive
+    Given I click on "Personal" "link"
+    When user "admin" uploads a file inside space "Personal" with content "some content" to "testfile.txt"
+    Then I should not see "testfile.txt"
+    But I should see "testfile.txt" after refreshing the file-picker
+
+  Scenario: click refresh button to get latest resource on Project drive
```

Comment:
Note: this is a sequence of two "tests". But the 2nd part follows directly after the first part, and depends on it anyway.
It could be split into 2 scenarios, but that would use up test run time without a real benefit. So, IMO, this is OK.